src/components/Transactions/TransactionContract.tsx

```diff
@@ -73,7 +73,7 @@ export const TransactionContract = ({ txDetails }: { txDetails: TxDetails }) =>
                   {...(txDetails.direction === 'inbound'
                     ? { color: 'green.500' }
                     : { color: 'inherit', prefix: '-' })}
-                  value={fromBaseUnit(txDetails.value, txDetails.precision)}
+                  value={fromBaseUnit(txDetails.value, txDetails.precision, 6, txDetails.negated)}
```

Comment:
I'd prefer if we didn't load up `useTxDetails` with `negated` - `TransactionContract` should have all of the info it needs to do the equivalent logic of `(tx.data.method === 'withdraw' && activeTx.type === 'send')`, and then modify the `prefix` accordingly.